test/commands/profiles/create.test.js

```diff
@@ -58,6 +59,33 @@ describe('commands', () => {
           );
         });
 
+      createTest()
+        .nock('https://api.twilio.com', api => {
+          api.get(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}.json`).reply(200, {
+            sid: constants.FAKE_ACCOUNT_SID
+          });
+          api.post(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}/Keys.json`).reply(200, {
+            sid: constants.FAKE_API_KEY,
+            secret: constants.FAKE_API_SECRET
+          });
+        })
+        .do(ctx => {
+          sinon.stub(os, 'hostname').returns('X'.repeat(64));
+          return ctx.testCmd.run();
+        })
+        .it('truncates apiKeyFriendlyName to 64 characters', ctx => {
```

Comment:
How do we know the name was truncated?

Reply:
So, that's the bit we struggled with yesterday. The issue is that the friendly name is sent in an api call to the back end, which we're mocking. I haven't quite figured out how to check the mocked GET request's payload before sending the mocked 200 response.

Reply:
I'd go with extracting out the friendly name creation (and truncation) to a separate method and just test that piece (no API mocking needed).

Reply:
updated!